tests/unit/async_/test_driver.py

```diff
@@ -136,7 +140,7 @@ async def test_routing_driver_constructor(protocol, host, port, params, auth_tok
             ConfigurationError, "The config settings"
         ),
         (
-            {"ssl_context": ssl.SSLContext(ssl.PROTOCOL_TLSv1)},
+            {"ssl_context": ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)},
```

Comment:
this seems unrelated?

Reply:
Caught :see_no_evil: It gets rid of a deprecation warning https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_TLSv1
I'll put it into it's own PR.

Reply:
https://github.com/neo4j/neo4j-python-driver/pull/780